frontend/src/container/FormAlertRules/ChartPreview/index.tsx

```diff
@@ -86,7 +86,8 @@ function ChartPreview({
 		{
 			query: query || initialQueriesMap.metrics,
 			globalSelectedInterval: selectedInterval,
-			graphType,
+			graphType:
```

Comment:
Why are we setting TimeSeries for the Bar panel type?

Reply:
@YounixM  The backend doesn't support graphTypes as `bar`, as we are consuming time series data for plotting bar chart, we have to send graphType as `time_series`. 

I added the comment now.